src/components/RecipeCard/RecipeCard.jsx

```diff
@@ -0,0 +1,63 @@
+import FontAwesomeIcon from '@fortawesome/react-fontawesome';
+import Like from 'components/Like/Like';
+import PropTypes from 'prop-types';
+import React from 'react';
+import UserChip from 'components/UserChip/UserChip';
+import { faClock } from '@fortawesome/fontawesome-free-regular';
+
+import './RecipeCard.scss';
+
+const RecipeCard = ({
+  title,
+  image,
+  cooktime,
+  description,
+  authorName,
+  authorImage,
+}) => (
+  <div className="recipe-card">
+    <div
+      className="recipe-card-image"
+      alt={title}
+      style={{ backgroundImage: `linear-gradient(to bottom, rgba(255,255,255,0) 20%, rgba(245, 242, 242, 1)), url(${image})` }}
```

Comment:
What are your thoughts on using `StyledComponents` to do this?

Reply:
Maybe not a great idea to use a lib for such a small thing but might be cleaner...

Reply:
https://www.styled-components.com/

Reply:
Yeah its a neat library that would make the code a bit cleaner but I'm not sure about adding a lib for this too..
@sukritchhabra how about we keep this in the back-pockets and file a backlog task. And if we have a lot of style injections like we are doing above then we can utilize the library for a cleaner way?